PWGEM/PhotonMeson/DataModel/gammaTables.h

```diff
@@ -56,13 +56,27 @@ DECLARE_SOA_DYNAMIC_COLUMN(RecalculatedVtxR, recalculatedVtxR, [](float x, float
 } // namespace gammarecalculated
 
 DECLARE_SOA_TABLE(V0Recalculated, "AOD", "V0RECALCULATED",
-                  o2::soa::Index<>,
-                  v0data::V0Id,
+                  //o2::soa::Index<>,
+                  //v0data::V0Id,
                   gammarecalculated::RecalculatedVtxX,
                   gammarecalculated::RecalculatedVtxY,
                   gammarecalculated::RecalculatedVtxZ,
                   gammarecalculated::RecalculatedVtxR<o2::aod::gammarecalculated::RecalculatedVtxX, o2::aod::gammarecalculated::RecalculatedVtxY>);
 
+namespace pdgcodetable
+{
+DECLARE_SOA_COLUMN(PdgCodeV0, pdgCodeV0, int); // PDG code of V0
```

Comment:
the PdgCode of the mc mother particle of the v0 (if its not a fake V0) is already part of McGammasTrue (along with some some mc information, maybe check the definition of McGammasTrue to see which columns it contains)

For the daughters, it would be better if you create a table for the mc daughter tracks, that contains both negative and positive daughter tracks. This has the advantage that whatever column you will want to save for the Mc daughter tracks, you will only have to define once and not twice.

Reply:
I changed the things as discussed. Maybe you can give it a look if you have some free time :D